lib/diagnostic.js

```diff
@@ -15,7 +15,16 @@ db.dropDatabase();
 // House Targaryen, motto 'Fire and Blood'
 
 // your code begins here
-
+db.houses.insert({
```

Comment:
These objects need to be in an array if you want to insert multiple documents at once.